src/test/java/teammates/test/pageobjects/InstructorCourseEditPage.java

```diff
@@ -7,12 +7,14 @@
 import org.openqa.selenium.WebElement;
 import org.openqa.selenium.support.FindBy;
 
+import teammates.common.util.Const;
+
 public class InstructorCourseEditPage extends AppPage {
     
-    @FindBy(id = "courseid")
+    @FindBy(id = Const.ParamsNames.COURSE_ID)
```

Comment:
I think the `Const.ParamsNames` actually corresponds to the `name=` attributes, not the `id=` attributes, it just so happens that it is the same for `courseid`. If you see below for `instructorname`, you'll that there is no guarantee that `id` attribute will be in sync with the `Const.ParamsNames`